pkg/authorization/api/v1/conversion_generated.go

```diff
@@ -836,9 +745,6 @@ func Convert_api_SubjectAccessReviewResponse_To_v1_SubjectAccessReviewResponse(i
 }
 
 func autoConvert_v1_SubjectRulesReviewStatus_To_api_SubjectRulesReviewStatus(in *SubjectRulesReviewStatus, out *authorization_api.SubjectRulesReviewStatus, s conversion.Scope) error {
-	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
```

Comment:
yep, I rewrote the defaulters in origin to match the kube... this was removed in upstream and you have to have individual defaulting functions that should not be anonymous anymore....